config/default/webhookcainjection_patch.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Should we remove this m/s?

Reply:
See https://github.com/rh-ecosystem-edge/kernel-module-management/pull/508/files#r1178887591